### PR TITLE
Added new campaigns, Updated existing campaigns

### LINKED ISSUE #16: Standardize the text in the tables
We have multiple texts denoting the same thing. Example: "pan india"-"national". Lets fix it to one or the other.

### PR DESCRIPTION
**Pull request by nileshbansod**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -74,216 +74,240 @@ While the citizen-led campaigns need your financial support and encouragement to
...
+         <td>Pan India</td>
+        </tr>
+        <tr class="   wage-workers  ">
+         <td>Zomato India <br /><span style="font-size:0.8em"> [<a href="https://www.zomato.com/blog/feed-daily-wager">Go To Page</a>] [<a href="#zomato-india">Details</a>]</span></td>
+         <td>Daily wage workers</td>
+         <td>Ration kits</td>
+         <td>Pan India</td>
+        </tr>
+        <tr class="     ">
+         <td>Kanaga (Personal Campaign) <br /><span style="font-size:0.8em"> [<a href="https://milaap.org/fundraisers/support-kanaga">Go To Page</a>] [<a href="#kanaga-(personal-campaign)">Details</a>]</span></td>
```
Comment (ayushbaid) [inline index.md]: fix this (no paranthesis)

File: index.md [hunk 2]
```diff
@@ -74,216 +74,240 @@ While the citizen-led campaigns need your financial support and encouragement to
...
+         <td>Delhi, Agra, Lucknow, Kanpur, Varanasi, Bangalore, Mumbai, Surat, Malegaon</td>
+        </tr>
+        <tr class="    homeless ">
+         <td>Uday Foundation <br /><span style="font-size:0.8em"> [<a href="https://www.udayfoundation.org/coronavirus-disease-covid-19/?fbclid=IwAR05051YKouzPYii14L2CNPnS-I80PLgvAj_tw4NUuFxcDIlGl8AS6ft7FA">Go To Page</a>] [<a href="#uday-foundation">Details</a>]</span></td>
+         <td>Homeless persons</td>
+         <td>Food, soap</td>
+         <td>Pan India</td>
+        </tr>
+        <tr class=" delhi  wage-workers  ">
+         <td>Delhi Youth Welfare Association <br /><span style="font-size:0.8em"> [<a href="https://www.ourdemocracy.in/Campaign/covid19Delhi">Go To Page</a>] [<a href="#delhi-youth-welfare-associaltion">Details</a>]</span></td>
```
Comment (ayushbaid) [inline index.md]: spelling mistake in association

File: index.md [hunk 3]
```diff
@@ -74,216 +74,240 @@ While the citizen-led campaigns need your financial support and encouragement to
...
+         <td>Ration kits</td>
+         <td>Mumbai</td>
+        </tr>
+        <tr class="  bangalore   trash-pickers">
+         <td>Hasiru Dala <br /><span style="font-size:0.8em"> [<a href="http://hasirudala.in/news/wastepickers-and-covid19/">Go To Page</a>] [<a href="#hasiru-dala">Details</a>]</span></td>
+         <td>Waste pickers</td>
+         <td>Ration kits</td>
+         <td>Bangalore, Mysuru, Tumakuru, Davanagere, Hubli, Dharawad</td>
+        </tr>
+        <tr class="     ">
```
Comment (ayushbaid) [inline index.md]: are you sure this was the intentional?

File: index.md [hunk 4]
```diff
@@ -74,216 +74,240 @@ While the citizen-led campaigns need your financial support and encouragement to
...
+         <td>Ration kits, cash</td>
+         <td>Gurgaon</td>
+        </tr>
+        <tr class=" delhi  wage-workers  ">
+         <td>KMC-Ramjas Collective <br /><span style="font-size:0.8em"> [<a href="https://pages.razorpay.com/pl_EXfaOUPkVTIPla/view">Go To Page</a>] [<a href="#kmc-ramjas-collective">Details</a>]</span></td>
+         <td>Daily wage workers, migrant workers</td>
+         <td>Ration kits, food, safety kits</td>
+         <td>Delhi</td>
+        </tr>
+        <tr class="     ">
```
Comment (ayushbaid) [inline index.md]: intentional?

**Review by ayushbaid — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (nileshbansod) [reply index.md]: Fixed

File: index.md (hunk 2 above)
Comment (nileshbansod) [reply index.md]: Fixed

File: index.md (hunk 3 above)
Comment (nileshbansod) [reply index.md]: Fixed

File: index.md (hunk 4 above)
Comment (nileshbansod) [reply index.md]: Fixed

File: index.md (hunk 2 above)
Comment (ayushbaid) [reply index.md]: still an issue

File: index.md [hunk 5]
```diff
@@ -301,113 +325,113 @@ Use the **All Campaigns** button to remove the filters and see all initiatives.
...
 Uday foundation is collecting donations to sponsor distribution of sanitization and food kits to homeless individuals and families. The kits will cover one month worth of supplies for individuals
 
-- City/States Covered: Pan India
-- Payment Modes Available: Card, Net Banking, PayTM
+- Cities/States Covered: Pan India
+- Payment Modes Available: Cards, Net Banking, Wallets
 - 80G Eligible: Yes
 
-### [Delhi Youth Welfare Association](https://www.ourdemocracy.in/Campaign/covid19Delhi)
+### [Delhi Youth Welfare Associaltion](https://www.ourdemocracy.in/Campaign/covid19Delhi)
```
Comment (ayushbaid) [inline index.md]: same issue

**Review by ayushbaid — CHANGES_REQUESTED** [review]

**Review by ayushbaid — APPROVED** [review]

**Pull request merged**